admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -979,6 +980,16 @@ var form = (function() {
       });
   }
 
+  function checkDOMChange() {
+    if($('.attribute_wholesale_price').length > 0 && $('.attribute_priceTE').length > 0) {
+      inputsTimeout = null;
+      $('.attribute_wholesale_price, .attribute_priceTE').each(function(){
+        $(this).val(priceCalculation.normalizePrice($(this).val()));
+      })
+    }else
+      inputsTimeout = setTimeout( checkDOMChange, 100 );
```

Comment:
Yes I know that this fix is not so clear but I did not manage to change the behavior in another way than that because the normalizePrice is called on the blur event of the input field.
The bug is very serious because it affects both the wholesale_price and the final combination's price causing the alteration of the final price of the combination. Could you please give me some ideas for an alternative solution?